src/main/java/uk/co/ramp/covid/simulation/place/householdtypes/LargeFamily.java

```diff
@@ -0,0 +1,62 @@
+package uk.co.ramp.covid.simulation.place.householdtypes;
+
+import uk.co.ramp.covid.simulation.place.Household;
+import uk.co.ramp.covid.simulation.population.Places;
+import uk.co.ramp.covid.simulation.util.RNG;
+
+/** LargeFamily: two adults of any age and three or more children,
+ *  or three or more adults of any age and one or more children */
+public class LargeFamily extends Household {
+
+    private boolean twoAdult = RNG.get().nextUniform(0,1) < 0.5;
```

Comment:
I think this parameter should go in the parameters file together with all the other parameters about household distributions (which probably means splitting this class into two classes).

Reply:
Agree this, and OlderSmaller, should probably be two classes